doc/RELEASE_WALKTHROUGH.rst.txt

```diff
@@ -99,25 +102,30 @@ may have been accessed and changed by someone else and a push will fail::
     $ git branch <new version>  # only when starting new numpy version
     $ git checkout v1.14.x  # v1.14.x already existed for the 1.14.4 release
 
-Edit the ``.travis.yml`` and ``.appveyor.yml`` files to make sure they have the
-correct version, and put in the commit hash for the ``REL`` commit created
-above for ``BUILD_COMMIT``, see the _example from `v1.14.3`::
+Edit the ``azure/posix.yml`` and ``azure/windows.yml`` files to make sure they
+have the correct version, and put in the commit hash for the ``REL`` commit
+created above for ``BUILD_COMMIT``, see an _example::
 
-    $ gvim .travis.yml .appveyor.yml
+    $ gvim azure/posix.yml azure/windows.yml
     $ git commit -a
     $ git push upstream HEAD
 
+Push and, if you are on a fork of the repo, merge the branch.
+
```

Comment:
This is confusing as to the meaning of "fork", "upstream" here always refers to the `MacPython/numpy-wheels` repository as stated at the beginning. This might be part of the getting setup for releasing notes. My `.git/config` file looks like
```
 ...
[remote "origin"]
	url = git@github.com:charris/numpy-wheels
	fetch = +refs/heads/*:refs/remotes/origin/*
[remote "upstream"]
	url = git@github.com:MacPython/numpy-wheels
	fetch = +refs/heads/*:refs/remotes/upstream/*
...
```
I mostly want clarification of what you intend to say. I plan to update the walkthrough after 1.19.0 is released using newer versions, etc.

Reply:
In your case the fork is "origin". So pushing a PR to "origin" will not upload the wheels, only when that fork is merged will it have the tokens to upload (as stated above).

Reply:
That is what I thought. But since the example pushes upstream I don't see the need to say this. At the beginning of the walkthrough there is this:
```
Note that in the code snippets below, ``upstream`` refers to the root repository on
github and ``origin`` to a fork in your personal account. You may need to make adjustments
if you have not forked the repository but simply cloned it locally. You can
also edit ``.git/config`` and add ``upstream`` if it isn't already present.
```
So this seems unnecessary and a bit confusing. Pushing to a branch upstream will initiate an upload, or should. Making a PR won't unless it is merged.

Reply:
removing